migrations/versions/ea5ad7587c7f_added_aliases_of_categories.py

```diff
@@ -0,0 +1,43 @@
+"""Added aliases of categories.
+
+Revision ID: ea5ad7587c7f
+Revises: f467ff99202e
+Create Date: 2020-03-20 16:04:46.594379
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+from migrations.util.add_field_data import add_field_data
+from models.Category import Category
+
+revision = 'ea5ad7587c7f'
+down_revision = 'f467ff99202e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    conn = op.get_bind()
+
+    op.add_column('category', sa.Column('category_alias', sa.String(length=30), nullable=False))
+    op.create_foreign_key(None, 'place', 'locale_string', ['place_name_id'], ['string_id'])
+
+    add_field_data(conn, Category.__table__, 'category_id', 1, 'category_alias', 'monuments')
```

Comment:
don't think that its good idea to add such data migrations, it will throw exception when executed on empty db. Lets add this data manually